.github/scripts/get_workflow_type.py

```diff
@@ -49,47 +48,34 @@ def is_exception_branch(branch: str) -> bool:
 
 
 def get_workflow_type(issue: Issue, username: str) -> str:
-    user_list = issue.get_comments()[0].body.split("\r\n")
     try:
-        run_option = issue.get_comments()[1].body.split("\r\n")[0]
-    except Exception as e:
-        run_option = "single"
-
-    if user_list[0] == "!":
-        # Use old runners for everyone
-        return WORKFLOW_TYPE_LABEL
-    elif user_list[1] == "*":
-        if run_option == WORKFLOW_TYPE_BOTH:
-            # Use ARC runners and old runners for everyone
-            return WORKFLOW_TYPE_BOTH
-        else:
-            # Use only ARC runners for everyone
-            return WORKFLOW_TYPE_RG
-    elif username in user_list:
-        if run_option == WORKFLOW_TYPE_BOTH:
-            # Use ARC runners and old runners for a specific user
-            return WORKFLOW_TYPE_BOTH
+        user_list = issue.get_comments()[0].body.split("\r\n")
+
+        if user_list[0] == "!":
+            return WORKFLOW_LABEL_META
+        elif user_list[1] == "*":
+            return WORKFLOW_LABEL_LF
+        elif username in user_list:
+            return WORKFLOW_LABEL_LF
         else:
-            # Use only ARC runners for a specific user
-            return WORKFLOW_TYPE_RG
-    else:
-        # Use old runners by default
-        return WORKFLOW_TYPE_LABEL
+            return WORKFLOW_LABEL_META
+    except Exception as e:
+        return WORKFLOW_LABEL_META
 
 
 def main() -> None:
     args = parse_args()
 
     if is_exception_branch(args.github_branch):
-        output = {"workflow_type": WORKFLOW_TYPE_LABEL}
+        output = {"label_type": WORKFLOW_LABEL_META}
     else:
         try:
             gh = get_gh_client(args.github_token)
```

Comment:
nit: can you add a comment linking to the default issue that is used here? It'll save people time searching for it and we're unlikely to need to use another issue anyways except possibly for testing